### ISSUE TITLE
Shell script error when building custom images

### ISSUE THREAD

**Issue by igor-susic** [GoogleCloudDataproc/custom-images#99]
Hi, 

Recently we started to receive errors when building custom data proc images with following the error message (the input parameters were changed from the real ones passed in):

````shell
+ python3 generate_custom_image.py --project-id EXAMPLE --image-name EXAMPLE --dataproc-version 2.2.6-ubuntu22 --customization-script ../build/cloud_image_modification_script.sh --extra-sources '{"requirements.txt": "../build/requirements.txt"}' --zone europe-west1-c --gcs-bucket EXAMPLE --storage-location europe-west1 --network EXAMPLE --subnetwork EXAMPLE --disk-size 30 --no-external-ip --service-account example@gdevexponea.iam.gserviceaccount.com
/tmp/tmpvf[88](Gitlab)k6tx: line 133: syntax error near unexpected token `then'
Traceback (most recent call last):
  File "/builds/example-project/custom-images/generate_custom_image.py", line 95, in <module>
    main()
  File "/builds/example-project/custom-images/generate_custom_image.py", line 88, in main
    shell_image_creator.create(args)
  File "/builds/example-project/custom-images/custom_image_utils/shell_image_creator.py", line 43, in create
    shell_script_executor.run(script)
  File "/builds/example-project/custom-images/custom_image_utils/shell_script_executor.py", line 47, in run
    raise RuntimeError("Error building custom image.")
RuntimeError: Error building custom image.

````

I suspect that the issue was introduced in recent MR:
* https://github.com/GoogleCloudDataproc/custom-images/commit/bdd715209557668e6a3a270a041ae47e80eabd8a#diff-9c5d2fb4f17dc7df741e9c63391616de2a30590e65d2bc5ac8e18f2c76a297a9R89

There is missing semicolon if I have interpreted it correctly:
* https://github.com/GoogleCloudDataproc/custom-images/blob/main/custom_image_utils/shell_script_generator.py#L157

@cjac could you verify this please. I can submit a change if needed. 

Thank you

**Comment by cjac** [comment]
oops, you're right.  That semicolon needs to be replaced.

**Comment by cjac** [comment]
Can you test the code from #102 and I'll merge once you've confirmed that it fixes the issue?

**Comment by cjac** [comment]
Fixed in #103
